### PR TITLE
[MINOR] Fix a bunch of typos

### PR DESCRIPTION
**Pull request by srowen**
## What changes were proposed in this pull request?

Fix a bunch of typos, to head off more typo PRs.

## How was this patch tested?

Existing tests

### CODE REVIEW

**Review by felixcheung — APPROVED** [review]
whoa

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master

**Review by jiangxb1987 — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master - https://github.com/apache/spark/commit/c284c4e1f6f684ca8db1cc446fdcc43b46e3413c

File: common/network-common/src/main/java/org/apache/spark/network/protocol/MessageWithHeader.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ public long transferred() {
    * transferTo invocations in order to transfer a single MessageWithHeader to avoid busy waiting.
    *
    * The contract is that the caller will ensure position is properly set to the total number
-   * of bytes transferred so far (i.e. value returned by transfered()).
+   * of bytes transferred so far (i.e. value returned by transferred()).
```
Comment (vanzin) [inline common/network-common/src/main/java/org/apache/spark/network/protocol/MessageWithHeader.java]: FYI the typo was actually in Netty's API, not here. (Netty 4.1 fixed it though.)

Comment (HyukjinKwon) [reply common/network-common/src/main/java/org/apache/spark/network/protocol/MessageWithHeader.java]: Thanks. I was there when we upgraded it to 4.1.17 ;).

**Pull request closed without merging**